src/Symfony/Component/Notifier/Bridge/Telegram/TelegramTransport.php

```diff
@@ -98,7 +98,7 @@ protected function doSend(MessageInterface $message): SentMessage
         if (200 !== $statusCode) {
             $result = $response->toArray(false);
 
-            throw new TransportException('Unable to post the Telegram message: '.$result['description'].sprintf(' (code %s).', $result['error_code']), $response);
+            throw new TransportException('Unable to post the Telegram message: '.$result['description'].sprintf(' (code "%s").', $result['error_code']), $response);
         }
```

Comment:
revert

Reply:
Why should I revert it? It has been shown on automatic test and it was written that I just pathc this (from fabpot bot) or something. So in this should I should skip it and revert this change?

Reply:
This is a false positive from fabbot, so it should be reverted.